src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -1,7 +1,41 @@
 package core.mate.academy.service;
 
-/**
- * Your implementation of MachineService.
- */
-public class MachineServiceImpl {
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import core.mate.academy.model.Truck;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MachineServiceImpl implements MachineService<Machine> {
+    private static final Class TYPE_BULLDOZER = Bulldozer.class;
+    private static final Class TYPE_EXCAVATOR = Excavator.class;
```

Comment:
in future afte hashMap you will learn how to handle such case with a strategy pattern, that solution will be flexible. 
For now these constants don't give us a lot of practical value, we can remove them